content/en/docs/languages/java/instrumentation.md

```diff
@@ -1185,9 +1185,10 @@ meter
 
 ## Logs
 
-Logs are distinct from Metrics and Tracing in that there is no user-facing logs
-API. Instead, there is tooling to bridge logs from existing popular log
-frameworks (e.g. SLF4j, JUL, Logback, Log4j) into the OpenTelemetry ecosystem.
+Logs are distinct from Metrics and Tracing in that **there is no user-facing
+OpenTelemetry logs API**. Instead, there is tooling to bridge logs from existing
+popular log frameworks (e.g. SLF4j, JUL, Logback, Log4j) into the OpenTelemetry
+ecosystem.
```

Comment:
```suggestion
ecosystem. To learn about the rationale behind this design decision, read the [Logging specification](/docs/specs/otel/logs/).
```

maybe giving people a pointer to read about the background of this?

Reply:
Maybe a bit more tersely:

> For rationale behind this design decision, see [Logging specification](/docs/specs/otel/logs/).

Reply:
```suggestion
ecosystem. For rationale behind this design decision, see [Logging specification](https://github.com/docs/specs/otel/logs/).
```